README.md

```diff
@@ -790,6 +790,24 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --npm-client-args [client]
```

Comment:
It seems accepting all args after `--` is the lottery winner. My only concern is it's not very explicit and bootstrap isn't just a proxy of `[npm | yarn] install` so it might be confusing whether args get appended to entire flow of bootstrap or particular parts of it. 

I can live with that :)